pyctcdecode/decoder.py

```diff
@@ -77,7 +87,7 @@ def _log_softmax(x: np.ndarray, axis: Optional[int] = None) -> np.ndarray:
     exp_tmp = np.exp(tmp)
     # suppress warnings about log of zero
```

Comment:
would it be better just to add epsilon=10**-100 to all values before taking logs?

Reply:
idk, this is taken straight from scipy